### ISSUE TITLE
Empty UL

### ISSUE THREAD

**Issue by timo54** [ATLAS-Illinois/Illinois-Framework-Theme#1082]
As an example, the "Amenities" page on the Illini Union Hotel website, (https://illiniunionhotel.illinois.edu/amenities) has an Unordered list immediately after the page's H1 title. This UL is empty and doesn't contain any LI elements. AInspector flags this as a violation and says, "Container widgets must have required owned elements." The UL should be commented out until it's needed to hold content. 

<img width="302" height="78" alt="Image" src="https://github.com/user-attachments/assets/0144a075-e744-4bac-8803-ffeb44cf8932" />

**Comment by trubach** [comment]
@nathanb-UIUC - Is this a site you manage? The theme's hero paragraph is set up to require at least one button, which in this instance, it doesn't have a button. The theme's twig is outputting the `<ul>` with the expectation that a button will be following. Did you change the settings for the hero paragraph to not require a  button?

**Comment by trubach** [comment]
Closing this issue since it's likely not an issue the IL Drupal Framework itself. Happy to reopen if it turns out it is.
